prboom2/src/p_enemy.c

```diff
@@ -1165,24 +1165,6 @@ void A_Look(mobj_t *actor)
   P_SetMobjState(actor, actor->info->seestate);
 }
 
-//
-// A_KeepChasing
-//
-// killough 10/98:
-// Allows monsters to continue movement while attacking
-//
-
-static void A_KeepChasing(mobj_t *actor)
```

Comment:
Lost boom compatibility?

Reply:
We probably don't want to delete any action functions, even if they aren't used.

Reply:
The compiler is going to simply omit this since it's unreferenced.  What should be using it?  Should it be a possible DEH code pointer?